### ISSUE TITLE
Skip to main content - put focus in search bar, show a secondary focus representation to show where skip link goes

### ISSUE THREAD

**Issue by maxkadel** [pulibrary/allsearch_frontend#483]
### Steps to recreate
1. Go to [allsearch](https://allsearch.princeton.edu/)
2. Tab to "Skip to main content"
3. It is unclear where the skip link will bring the user
4. Hit enter to select the skip link
5. The link actually brings focus to the section that surrounds the search box, and does not put user focus inside the search box (you cannot immediately start typing), contravening user expectations.

This stems from a Fable user test - the tester uses a quad stick, and interacts with it mainly through keyboard navigation.

> Oh, center skip to main content. Just as good. I'm assuming if I hit enter there, my focus would jump to the search bar. But I will say, that's not clear, and I always suggest a second focus representation telling me where exactly, and highlighting where exactly I'm going to jump to. 

### Severity
Moderate

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
